src/pages/snapshotTimes/index.tsx

```diff
@@ -152,11 +152,14 @@ function App() {
   } else {
     return (
       <>
-        <Center>
-          <Text fontSize="3xl" margin="20px">
-            time before gauge vote start snapshot was taken
-          </Text>
-        </Center>
+        <Text
```

Comment:
hopefully that guy that says he is working on the theme gets it done. it would be great to have a variant of <Text> we could just use places that would be automatically responsive.